docker-compose.yml

```diff
@@ -235,6 +270,32 @@ services:
     networks:
       - callisto
 
+  person-restapi:
+    image: "quay.io/ukhomeofficedigital/callisto-person-restapi:$PERSON_RESTAPI_TAG"
+    environment:
+      <<: *db-env
+      BOOTSTRAP_SERVER: kafka:9093
+      KEYSTORE_LOCATION: file://keystore/person-restapi/person-restapi.keystore.jks
```

Comment:
I am not sure this would work. The only reference of `KEYSTORE_LOCATION` environment variable in timecard and person projects is in `docker-compose.yaml` file, which is used for local development only. Meaning that the default has to match the keystore location set in the deployment.yaml

Reply:
Following a conversation with @taomoh we decided that a better time to address this would be when we look to retrofit a common helm chart to our timecard & person rest api services

Reply:
Ticket created for this: https://collaboration.homeoffice.gov.uk/jira/browse/EAHW-2701